packages/pluggableWidgets/maps-native/src/Maps.editorConfig.ts

```diff
@@ -74,7 +74,7 @@ export function getProperties(values: MapsPreviewProps, defaultProperties: Prope
     return defaultProperties;
 }
 
-export function check(values: MapsPreviewProps): Problem[] {
+export function check(values: Omit<MapsPreviewProps, "class" | "style">): Problem[] {
```

Comment:
We don't have them here?

Reply:
Its not necessary for the test, so i just omitted. I could as well just added the className, style and readOnly but they are irrelevant for native.